pandas/io/tests/test_common.py

```diff
@@ -33,7 +33,8 @@ class TestCommonIOCapabilities(tm.TestCase):
 foo2,12,13,14,15
 bar2,12,13,14,15
 """
-
+    data2 = data1.replace(",", " ")
+    
     def test_expand_user(self):
```

Comment:
I think your original example was fine.  No need to do this here unless `data2` is intended to be used elsewhere.  All you had to do was change `read_table` to `read_csv`.